src/fetch_tools/commands/debug_snapshot.py

```diff
@@ -35,8 +35,8 @@
 # All commands requiring sudo must be added in main
 commands = {"dpkg_fetch":"COLUMNS=200 dpkg -l ros-indigo-fetch-*",
             "dpkg_all":"COLUMNS=200 dpkg -l",
```

Comment:
Saw the COLUMNS trick here for the first time... Question: was COLUMNS=200 set to reduce the size of the logs? 

(If it was set to make sure the output was wide enough, I'm getting a width of >400 by default when using dpkg via e.g. subprocess.check_output, so it's probably wide enough to begin.)